### PR TITLE
Fix aws ec2 describe instances

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-8489](https://jira-hq.paloaltonetworks.local/browse/CIAC-8489)

## Description
Fixed an issue where the **filter** argument for **aws-ec2-describe-instances** command didn't filter results as expected.

**No mandatory dependencies were added on this pull request.**

### CODE REVIEW

File: Packs/AWS-EC2/ReleaseNotes/1_2_39.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### AWS - EC2
+
+- Fixed an issue where the **filter** argument for **aws-ec2-describe-instances** command didn't filter results as expected.
```
Comment (JudahSchwartz) [inline Packs/AWS-EC2/ReleaseNotes/1_2_39.md:6]:
```suggestion
- Fixed an issue where the **filter** argument for **aws-ec2-describe-instances** command did not filter results as expected.
```
Lets say excluded the ':' character when it shouldnt have

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml [hunk 2]
```diff
@@ -3712,17 +3712,17 @@ script:
...
     - description: The IPv6 CIDR range. You can either specify a CIDR range or a source security group, not both. To specify a single IPv6 address, use the /128 prefix length.
       name: Ipv6RangesCidrIp
-    - description: 'A description for the security group rule that references this IPv6 address range. Constraints: Up to 255 characters in length. Allowed characters are a-z, A-Z, 0-9, spaces, and ._-:/()#,@[]+=&;{}!$*'
+    - description: 'A description for the security group rule that references this IPv6 address range. Constraints: Up to 255 characters in length. Allowed characters are a-z, A-Z, 0-9, spaces, and ._-:/()#,@[]+=&;{}!$*.'
       name: Ipv6RangesDescription
     - description: The ID of the prefix.
       name: PrefixListId
-    - description: 'A description for the security group rule that references this prefix list ID. Constraints: Up to 255 characters in length. Allowed characters are a-z, A-Z, 0-9, spaces, and ._-:/()#,@[]+=;{}!$*'
+    - description: 'A description for the security group rule that references this prefix list ID. Constraints: Up to 255 characters in length. Allowed characters are a-z, A-Z, 0-9, spaces, and ._-:/()#,@[]+=;{}!$*.'
```
Comment (JudahSchwartz) [inline Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:3723]: the period here is weird. Maybe we should delete the 1 in the beginning? But this is a weird way to satisfy the period rule

Comment (YuvHayun) [reply Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:3723]: I'll just remove it.

Comment (YuvHayun) [reply Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:3723]: I'll just remove it so it won't confuse customers.

**Review by JudahSchwartz — APPROVED** [review]

**Comment by YuvHayun** [comment]
@ShahafBenYakir need force-merge on this PR, pr is failing on TPB execution due to several missing variables aws variables which can't be created atm due to permissions issue.. a ticket to the lab was open [here](https://jira-hq.paloaltonetworks.local/browse/CRTX-94759).
I'll correct the TPB once the lab ticket will be solved.

**Pull request merged**